src/Layer/LabelLayer.js

```diff
@@ -1,17 +1,134 @@
 import * as THREE from 'three';
 import LayerUpdateState from 'Layer/LayerUpdateState';
 import ObjectRemovalHelper from 'Process/ObjectRemovalHelper';
-import Layer from 'Layer/Layer';
+import GeometryLayer from 'Layer/GeometryLayer';
 import Coordinates from 'Core/Geographic/Coordinates';
 import Extent from 'Core/Geographic/Extent';
 import Label from 'Core/Label';
 import { FEATURE_TYPES } from 'Core/Feature';
 import { readExpression } from 'Core/Style';
+import { ScreenGrid } from 'Renderer/Label2DRenderer';
 
 const coord = new Coordinates('EPSG:4326', 0, 0, 0);
 
 const _extent = new Extent('EPSG:4326', 0, 0, 0, 0);
 
+const nodeDimensions = new THREE.Vector2();
+const westNorthNode = new THREE.Vector2();
+const labelPosition = new THREE.Vector2();
+
+/**
+ * DomNode is node of tree data structure for labels div.
```

Comment:
I would say `DomNode is a node in the tree data structure of labels divs`

Reply:
yes